content/concepts/components.md

```diff
@@ -7,10 +7,16 @@ Before reading this page, you should understand some [Ocean-specific terminology
 
 ## Aquarius
 
-Marketplaces run Aquarius to store and manage metadata about the [assets](/concepts/terminology/#asset-or-data-asset) available in their marketplace. It provides an HTTP API for interacting with an off-chain database ([OceanDB](/concepts/components/#oceandb-drivers)).
+Marketplaces run Aquarius to store and manage metadata about the [assets](/concepts/terminology/#asset-or-data-asset) available in their marketplace. It provides an HTTP API for interacting with an off-chain database (sometimes called "OceanDB").
 
 <repo name="aquarius"></repo>
 
+Aquarius supports several options for the off-chain database, including MongoDB, Elasticsearch and BigchainDB. One can add support for another off-chain database by creating a new driver similar to the existing OceanDB drivers.
```

Comment:
would structure this with a subheading `### OceanDB`, and further down `### Osmosis` before the osmosis repo list text. Increases ability to scan page quickly and also makes it show up in TOC

Reply:
Okay I will add subheadings.